src/arch/arm/ArmSemihosting.py

```diff
@@ -33,13 +33,14 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from m5.objects.BaseSemihosting import BaseSemihosting
 from m5.objects.Serial import SerialDevice
 from m5.objects.Terminal import Terminal
 from m5.params import *
 from m5.SimObject import *
 
 
-class ArmSemihosting(SimObject):
+class ArmSemihosting(BaseSemihosting):
```

Comment:
Please remove the parameters that are specified in the `BaseSemihosting`

Reply:
Good catch, will update